### ISSUE TITLE
question about FID

### ISSUE THREAD

**Issue by ZHEGG** [lucidrains/denoising-diffusion-pytorch#226]
I use flowers dataset, but find although the picture is getting clearer, there is no significant decline in FID indicators, So I wonder how much reference FID really means?

my settings are as below:

import torch
import os
from denoising_diffusion_pytorch import Unet, GaussianDiffusion, Trainer

model = Unet(dim=64,dim_mults=(1,2,4,8),channels=3)

diffusion = GaussianDiffusion(model,image_size=256,timesteps=1000,sampling_timesteps=250)

image_folder = os.path.join('./dataset/flowers')
trainer = Trainer(diffusion,image_folder,train_batch_size=18,train_lr=8e-5,train_num_steps=700000,gradient_accumulate_every=2,ema_decay=0.995,amp=False,calculate_fid=True,convert_image_to="RGB")

trainer.train()

and FID remains around 160

when 1K iter 
![sample-1](https://github.com/lucidrains/denoising-diffusion-pytorch/assets/65839993/867fdb64-5794-4f19-96b2-0fab89866339)

and when 9k iter
![sample-9](https://github.com/lucidrains/denoising-diffusion-pytorch/assets/65839993/d7598107-9411-4917-b4af-2b1550a94a59)

**Comment by lhaippp** [comment]
Try to generate as much data as the training set and then calculate the fid

**Comment by Adversarian** [comment]
I believe this is addressed in #224. I was facing a similar problem which is caused by the fact that the current FID implementation uses very few samples to provide a meaningful metric. I am working on submitting a PR that resolves this issue.
